selfhost/main.jakt

```diff
@@ -154,6 +154,7 @@ function main(args: [String]) {
         let basename_without_extension = file_path.basename().substring(start: 0, length: file_path.basename().length() - 5)
         let cpp_filename = binary_dir + "/" + basename_without_extension + ".cpp"
         let output_filename = binary_dir + "/" + basename_without_extension
+        mkdir(binary_dir.c_string(), 0)
```

Comment:
this 0 allows it to compile, but I need to figure out how to represent something like `0777`. also the extern function in the prelude calls the parameter a `c_int` which might be wrong.